src/test/java/org/apache/drillui/test/framework/steps/webui/BaseSteps.java

```diff
@@ -72,4 +77,19 @@ public static void waitForURL(String url) {
   public static String getURL() {
     return WebBrowser.getURL();
   }
+
+  protected WebDriver getDriver() {
+    return WebBrowser.getDriver();
+  }
+
+  protected static void pressEsc() {
```

Comment:
I think it would be better to change this method to the more universal **pressKey(int keyCode)**

Reply:
Done.